R/sensitivity-analysis-run-options.R

```diff
@@ -10,16 +10,16 @@ SensitivityAnalysisRunOptions <- R6::R6Class(
   public = list(
     #' @description
     #' Initialize a new instance of the class
-    #' @param numberOfCoresToUse Number of cores to use for the simulation. Default value is `ospsuiteEnv$numberOfCoresToUse`
+    #' @param numberOfCores Number of cores to use for the simulation. Default value is `ospsuiteEnv$numberOfCores`
     #' @param showProgress Should a progress information be displayed. Default value is `ospsuiteEnv$showProgress`
     #' @return A new `SensitivityAnalysisRunOptions` object.
-    initialize = function(numberOfCoresToUse = ospsuiteEnv$numberOfCoresToUse,
+    initialize = function(numberOfCores = ospsuiteEnv$numberOfCores,

```

Comment:
@abdullahhamadeh I have renamed the `numberOfCoresToUse` to `numberOfCores`. This is a breaking change but I want to make sure that we have a consistent nomenclature in our tools.